test/test-execute/exec-supplementarygroups-single-group-user.service

```diff
@@ -2,7 +2,7 @@
 Description=Test for Supplementary Group with only one group and uid 1
 
 [Service]
-ExecStart=/bin/sh -x -c 'test "$$(id -G)" = "1" && test "$$(id -g)" = "1" && test "$$(id -u)" = "1"'
+ExecStart=/bin/sh -x -c 'test "$$(id -g)" = "1" && test "$$(id -u)" = "1" && ! id -G | grep 0'
```

Comment:
@tixxdz , I think we should remove all user/group tests from the `test-execute`. I'm not sure we can reliable test this on the host. But we can patch the user database in QEMU/nspawn-tests